ListBuilder/DynamicListBuilder.php

```diff
@@ -125,12 +125,13 @@ private function getDownloadUrl(): string
         if (null === $this->downloadUrl) {
             // The given id must be a number which we replace
             $idReplacerNumber = '875421';
+            $localeToReplace = 'en';
```

Comment:
use `{locale}` or `__locale__` here:


```suggestion
            $localeToReplace = '{locale}';
```

The `en` is not unique enough here. If somebody change the admin url to a domain which may contains `en` letters or a directory it would replace that also. So you need something more unique like `{locale}`.

you might still need str replace url encoded version of $localeToReplace not sure if symfony router use raw url encode or url encode.

Reply:
ping @martinlagler :)

Reply:
@alexander-schranz Ups 😅  fixed now.